Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -635,6 +635,10 @@ open class AVFoundationPlayback: Playback {
         #endif
     }
 
+    open override func changeSubtitle(style textStyle: [TextStyle]) {
+        self.player?.currentItem?.textStyle = textStyle
```

Comment:
This method has a currentItem nullable as a dependency and it is possible to be called without a currentItem available, this way this method will do nothing silently what could cause a semantical problem. Maybe a possible solution would be returns a bool or an exception when the dependency was not attended.